### PR TITLE
Fix time zone compatibility

### PR DESCRIPTION
**Pull request by HangyuanLiu**
Fix time zone compatibility with old fe

### CODE REVIEW

File: gensrc/thrift/PaloInternalService.thrift [hunk 1]
```diff
@@ -182,7 +182,8 @@ struct TPlanFragmentExecParams {
 struct TQueryGlobals {
   // String containing a timestamp set as the current time.
   1: required string now_string
-  2: optional string time_zone
+  2: optional i64 timestamp
```
Comment (imay) [inline gensrc/thrift/PaloInternalService.thrift]: Add some comments to describe timestamp and now_string's relation
```suggestion
  2: optional i64 timestamp_ms
```

File: be/src/runtime/runtime_state.cpp [hunk 2]
```diff
@@ -169,11 +177,19 @@ Status RuntimeState::init(
     const TQueryGlobals&  query_globals, ExecEnv* exec_env) {
     _fragment_instance_id = fragment_instance_id;
     _query_options = query_options;
-    _timestamp = atol(query_globals.now_string.c_str());
     if (query_globals.__isset.time_zone) {
         _timezone = query_globals.time_zone;
+        _timestamp_ms = query_globals.timestamp;
+    } else if (query_globals.now_string != "") {
```
Comment (imay) [inline be/src/runtime/runtime_state.cpp]: can this string be empty? I think it is no need to test it

**Review by imay — CHANGES_REQUESTED** [review]

**Review by imay — DISMISSED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**